ci-operator/config/openshift/openstack-test/openshift-openstack-test-main.yaml

```diff
@@ -68,6 +68,9 @@ tests:
   optional: true
   steps:
     cluster_profile: openstack-vh-mecha-az0
+    env:
```

Comment:
Is there a plan to also run it on periodics?

Reply:
yes when this whole thing is stable, we will have it in periodics.